storage/Devices/BcacheCset.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2016 SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact Novell, Inc.
+ *
+ * To contact Novell about this file by physical or electronic mail, you may
+ * find current contact information at www.novell.com.
+ */
+
+
+#ifndef STORAGE_BCACHE_CSET_H
+#define STORAGE_BCACHE_CSET_H
+
+
+#include "storage/Devices/Device.h"
+
+
+namespace storage
+{
+
+
+    //! A Bcache cache set.
```

Comment:
A short explanation of the relation between Bcache and BcacheSet? Like, which one is the SSD and which one hosts the FS on a spinning disk.

It is probably written in the upstream docs, but even the README manages to avoid the issue:
```
You'll need make-bcache from the bcache-tools repository. Both the cache device
and backing device must be formatted before use.
  make-bcache -B /dev/sdb
  make-bcache -C /dev/sdc
```